.idea/compiler.xml

```diff
@@ -6,5 +6,6 @@
         <outputRelativeToContentRoot value="true" />
       </profile>
     </annotationProcessing>
+    <bytecodeTargetLevel target="1.8" />
```

Comment:
We could remove the `.idea` folder and add it to the .`gitignore` file.

Reply:
removed all the .idea folder and added it to git ignore